src/widgets/RRule.tsx

```diff
@@ -0,0 +1,249 @@
+import * as React from 'react';
+import Container from './Container';
+import { TextField, Select, MenuItem, FormGroup, FormControlLabel, Checkbox } from '@material-ui/core';
+import RRule, { Options, Weekday } from 'rrule';
+
+import DateTimePicker from '../widgets/DateTimePicker';
+
+interface PropsType {
+  onChange: (rrule: string) => void;
+  rrule: string;
+}
+const frequency = {
+  yearly: 0,
+  monthly: 1,
+  weekly: 2,
+  daily: 3,
+};
+const weekdays = { 0: 'MO', 1: 'TU', 2: 'WE', 3: 'TH', 4: 'FR', 5: 'SA', 6: 'SU' };
+const months = {
```

Comment:
Same with this, it probably exists somewhere, but anyhow, you can at least make it an enum which would be much simpler/cleaner:

enum Months {
    Jan = 1,
    Feb,
    Mar,
    ...
}